### PR TITLE
fix(FTL-8111): fixed dev environment for Cognito

### PR DESCRIPTION
**Pull request by oleksiipiliugin**
#### sdk/core 7.11.0
* Cognito tests users were created on a new `dev` cloud environment;
* Integration tests secretes were updated according to the new dev environment;
* [Changed](https://github.com/affinidi/affinidi-core-sdk/pull/278/files#diff-40fba19984ea8072cc29f5f56eddfb9472d36c79f47559453b36b3b59e49c7c6R3) default Cognito config to `dev`;
* [Changed](https://github.com/affinidi/affinidi-core-sdk/pull/278/files#diff-369899346bb28dc6eff457d413e6b6856a6ac64de8ebd854a2217750225efcaeR16) default environment for tests to `dev`;
* [Adjusted](https://github.com/affinidi/affinidi-core-sdk/pull/278/files#diff-ec89872b3b37cd97ede1804f3cc36d0a2ce5c28672ee35b5e093ee5a42c2da19R27) OTP tests flows to new custom message whitelist approach;
* [Bumped](https://github.com/affinidi/affinidi-core-sdk/pull/278/files#diff-0c553b8004e94d39db471b45f6452b049ac538ff18f2b8eca54b0632b99e6787R44) `internal-api-clients`, `affinidi-did-auth-lib`, `user-management`, `common` versions.
#### sdk/node 7.10.1
* [Fixed](https://github.com/affinidi/affinidi-core-sdk/pull/278/files#diff-c8b4c3788483a78e4d57a526acdeceb7e389add00ed1859c5199975411089f60R42) `jsonld-signatures-bbs` version to `1.1.0` because of lib breaking changes.
#### libs/internal-api-clients 2.5.0
* [Updated](https://github.com/affinidi/affinidi-core-sdk/pull/278/files#diff-0147669e05fe91c4f855cb8edc2be2be1c12131e36c26fcc3fb8489e5d180304R13) `getSignedCredential` endpoint for `Keystorage` service.

#### sdk/browser 7.9.1
* [Bumped](https://github.com/affinidi/affinidi-core-sdk/pull/278/files#diff-8df48c8a2d6b0b97007c75ce36aeeb9030e8991fe3d637cfecadb95c83ac4cc2R41) `core` version. bumped `common` lib version in dev dependencies.
#### sdk/expo 7.10.1
* [Bumped](https://github.com/affinidi/affinidi-core-sdk/pull/278/files#diff-d93f82ba62318b64d08d32ce5302be1ae26e55f66a7c9b9eeaa67c81f1e18642R42) `core` version. bumped `common` lib version in dev dependencies.
#### sdk/react-native 7.10.1
* [Bumped](https://github.com/affinidi/affinidi-core-sdk/pull/278/files#diff-d3186c0aff9f996c5fa096d073674f9ccbeaa0a306e313fde29d155f935598a5R35) `core` version. bumped `common` lib version in dev dependencies.
#### libs/user-management 2.3.1
* [Bumped](https://github.com/affinidi/affinidi-core-sdk/pull/278/files#diff-7244df8614894b5fcb313cbf7c3fde91128082d527b71829c4681a7b71503c56R54) `internal-api-clients`, `common` versions.
#### libs/common 2.11.1
* [Bumped](https://github.com/affinidi/affinidi-core-sdk/pull/278/files#diff-fa5e37a32534210253b0d56428fd6aa6edbe6dfc084243ece2383d5f091b760dR46) `internal-api-clients` version.
#### libs/did-auth-lib 2.5.3
* [Bumped](https://github.com/affinidi/affinidi-core-sdk/pull/278/files#diff-3d765875167d0a1d6b97080fda8ddadf2dfbe7e33c91ed4442605d01d3d555aaR45) `internal-api-clients`, `common` versions.

### CODE REVIEW

File: common-libs/common/CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-# release 2.11.0 (2022-12-07)
+# release 2.11.1 (2023-01-20)
+* bumped `internal-api-clients` version
+* # release 2.11.0 (2022-12-07)
```
Comment (anton-iskryzhytskyi) [inline common-libs/common/CHANGELOG.md:3]:
```suggestion
# release 2.11.0 (2022-12-07)
```
let's keep our changelog in good state

**Comment by anton-iskryzhytskyi** [comment]
image
what the reason for this amount of changes in pkg-lock?

File: common-libs/internal-api-clients/src/services/KeyStorageApiService.ts [hunk 2]
```diff
@@ -10,7 +10,7 @@ type GetSignedCredentialOptions = {
   // For remote IssuerApiService (used by networkMember.verifyCredentialOfferResponseToken)
   env?: Env
   issuerUrl?: string
-  accessApiKey?: string
+  accessApiKey: string
```
Comment (anton-iskryzhytskyi) [inline common-libs/internal-api-clients/src/services/KeyStorageApiService.ts:13]: As I remember, any call to this service require apiKey, so it is not breaking change

File: sdk/browser/CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,6 @@
-# release 7.9.0 (2022-12-19)
+# release 7.9.1 (2023-01-20)
+* bumped `core` version. bumped `common` lib version in dev dependencies
+* # release 7.9.0 (2022-12-19)
```
Comment (anton-iskryzhytskyi) [inline sdk/browser/CHANGELOG.md:3]:
```suggestion
# release 7.9.0 (2022-12-19)
```

File: sdk/expo/CHANGELOG.md [hunk 4]
```diff
@@ -1,4 +1,6 @@
-# release 7.10.0 (2022-12-19)
+# release 7.10.1 (2023-01-20)
+* bumped `core` version. bumped `common` lib version in dev dependencies
+* # release 7.10.0 (2022-12-19)
```
Comment (anton-iskryzhytskyi) [inline sdk/expo/CHANGELOG.md:3]:
```suggestion
# release 7.10.0 (2022-12-19)
```

**Review by anton-iskryzhytskyi — CHANGES_REQUESTED** [review]
LGTM except some small but important things (including changelogs and pkg-json changes), please check my comments

**Comment by oleksiipiliugin** [comment]
> image
> 
> what the reason for this amount of changes in pkg-lock?

The reason is indentation changes.

**Review by yaroslava-kurash — APPROVED** [review]

**Review by anton-iskryzhytskyi — APPROVED** [review]

**Pull request merged**